extensions/interactions/rule_specs_html_mapping.json

```diff
@@ -0,0 +1,41 @@
+{
+  "ListOfSetsOfHtmlStrings": {
+    "interactionId": "DragAndDropSortInput",
+    "ruleTypes": {
+      "IsEqualToOrdering": {
+        "inputVariable": ["x"]
```

Comment:
Should be inputVariables because it's a list, right?

Or maybe htmlInputVariables?

Reply:
done